### DISCUSSION TITLE
Option to disable side lighting in latency certifier

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#19387] [Ideas] [answered]
When I was doing the latency certifier test in lazer, I found the side lighting to be very distracting and I feel like there should be some sort of option to disable it. In a test like this when you're trying to focus really hard on seeing the difference between both sides, having that side lighting there feels like it makes the test artificially harder than it's supposed to be for no good reason.

![osu_2022-07-25_21-57-22](https://user-images.githubusercontent.com/74088315/180913471-3b4d97f4-c78f-4198-b2a7-474616e25a05.png)

**Comment by peppy** [comment, marked as answer]
The idea behind the lighting (and the hidden area at the centre) is to avoid users nitpicking and doing what you seem to be doing. My hope with this mode is to let the user A B test based on perception alone, and not being able to cross-reference too closely. Hopefully that makes sense.

**Reply by Flamiii** [reply]
That makes sense. Maybe having something like a 10-15 second timer for each round could ensure that users who are certified at any given level are able to naturally distinguish latency with a certain degree of confidence as a result of their perception as opposed to just nitpicking.

**Reply by peppy** [reply]
A timer may be nice, definitely.
